src/ECBConverter.php

```diff
@@ -43,7 +43,7 @@ public function checkFileCache(): void
         if (!file_exists($this->cache_file) || time() - filemtime($this->cache_file) > $this->cache_timeout) {
             $this->reloadExchangeReferences();
             file_put_contents($this->cache_file, serialize($this->exchange_data), LOCK_EX);
-        } elseif (is_null($this->exchange_data)) {
+        } else if (is_null($this->exchange_data)) {
```

Comment:
@Richardds  Why did you change this?

Reply:
Ah, you're right. :D Although [both are valid](https://www.php.net/manual/en/control-structures.elseif.php).

https://github.com/Richardds/ECB-API/commit/e0877e691de3d2a72cfa309f782fa8c0ab0c75c1